cmd/snap/cmd_run.go

```diff
@@ -160,6 +168,13 @@ func (x *cmdRun) Execute(args []string) error {
 	if len(args) == 0 {
 		return fmt.Errorf(i18n.G("need the application to run as argument"))
 	}
+	if x.Explain || osutil.GetenvBool("SNAP_EXPLAIN") {
+		explain.Enable()
+	}
+	explain.Say("The extra output from snap run --explain is not suitable for parsing.")
+	explain.Header("snap run")
+	explain.Say("Invoked as %q", os.Args[0])
```

Comment:
this should say Invoked as "snap run",  if os.Args[0] == "snap"   , invoked as "snap" is too terse (too many things are snap in our universe)